docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -447,7 +469,7 @@ This flag will only send the header Authorization with a type of "Basic." If a r
 
 ### `--tunnel-capath`
```

Comment:
The sections for `--tunnel-capath` and `--capath` should be removed as well:
<img width="885" alt="Screenshot 2023-05-05 at 7 21 45 PM" src="https://user-images.githubusercontent.com/15352964/236583996-8cf74d7d-65c6-4265-9563-bedc1fed4efb.png">